bin/x86/bin/install_script.sh

```diff
@@ -0,0 +1,153 @@
+	fi
+fi
+
+DEFAULT_INSTALL_DIR="/opt/clanton-tiny/1.4.2"
+SUDO_EXEC=""
+target_sdk_dir=""
+answer=""
+relocate=1
+savescripts=0
+verbose=0
+while getopts ":yd:DRS" OPT; do
+	case $OPT in
+	y)
+		answer="Y"
+		[ "$target_sdk_dir" = "" ] && target_sdk_dir=$DEFAULT_INSTALL_DIR
+		;;
+	d)
+		target_sdk_dir=$OPTARG
+		;;
+	D)
+		verbose=1
+		;;
+	R)
+		relocate=0
+		savescripts=1
+		;;
+	S)
+		savescripts=1
+		;;
+	*)
+		echo "Usage: $(basename $0) [-y] [-d <dir>]"
+		echo "  -y         Automatic yes to all prompts"
+		echo "  -d <dir>   Install the SDK to <dir>"
+		echo "======== Advanced DEBUGGING ONLY OPTIONS ========"
+		echo "  -S         Save relocation scripts"
+		echo "  -R         Do not relocate executables"
+		echo "  -D         use set -x to see what is going on"
+		exit 1
+		;;
+	esac
+done
+
+if [ $verbose = 1 ] ; then
+	set -x
+fi
+
+absolute_sdk_dir=$(readlink -f "$0")
+absolute_sdk_dir=`dirname "$absolute_sdk_dir"`
+cd "$absolute_sdk_dir"
+
+target_sdk_dir=$(pwd)
+# Try to create the directory (this will not succeed if user doesn't have rights)
+mkdir -p $target_sdk_dir >/dev/null 2>&1
+
+# if don't have the right to access dir, gain by sudo
+if [ ! -x "$target_sdk_dir" -o ! -w "$target_sdk_dir" -o ! -r "$target_sdk_dir" ]; then
+	SUDO_EXEC=$(which "sudo")
+	if [ -z $SUDO_EXEC ]; then
+		echo "No command 'sudo' found, please install sudo first. Abort!"
+		exit 1
+	fi
+
+	# test sudo could gain root right
+	$SUDO_EXEC pwd >/dev/null 2>&1
+	[ $? -ne 0 ] && echo "Sorry, you are not allowed to execute as root." && exit 1
+
+	# now that we have sudo rights, create the directory
+	$SUDO_EXEC mkdir -p $target_sdk_dir >/dev/null 2>&1
+fi
+
+if [ -e "$target_sdk_dir/prev_install" ]; then
+    old_installed_dir=$($SUDO_EXEC cat "$target_sdk_dir/prev_install")
+
+    # installation is done already
+    if [ "$target_sdk_dir" == "$old_installed_dir" ]; then
+         echo done
+         exit
+    fi
+fi
+
+if [ ! -e "$target_sdk_dir/environment-setup-i586-poky-linux-uclibc" ]; then
+printf "Extracting SDK..."
+cat linux-clanton-tiny-uclibc-i586-i586-toolchain-1.4.2.tar.bz2 | $SUDO_EXEC tar xj -C $target_sdk_dir
+echo "done"
+fi
+printf "Setting it up..."
+env_setup_script="$target_sdk_dir/environment-setup-i586-poky-linux-uclibc"
+# fix environment paths
+#for env_setup_script in `ls $target_sdk_dir/environment-setup-*`; 
+	$SUDO_EXEC sed -e "s:$DEFAULT_INSTALL_DIR:$target_sdk_dir:g" -i "$env_setup_script"
+#done
+
+# fix dynamic loader paths in all ELF SDK binaries
+native_sysroot=$($SUDO_EXEC cat "$env_setup_script" |grep OECORE_NATIVE_SYSROOT|cut -d'=' -f2|tr -d '"')
+dl_path=$($SUDO_EXEC find "$native_sysroot"/lib -name "ld-linux*")
+if [ "$dl_path" = "" ] ; then
+	echo "SDK could not be set up. Relocate script unable to find ld-linux.so. Abort!"
+	exit 1
+fi
+#executable_files="'"$($SUDO_EXEC find "$native_sysroot" -type f -perm +111)"'"
+executable_files=$($SUDO_EXEC find "$native_sysroot" -type f -perm +111 -exec printf "\"%s\" " {} \; )
```

Comment:
Yeah. this was checked in by mistake when I pulled over the sketch upload tools.